Frontend/ui-customer/src/Screens/TicketingScreen/View.purs

```diff
@@ -0,0 +1,325 @@
+                , gravity BOTTOM
+                , weight 1.0
+                ]
+                [ 
+                  -- NavBar.view (push <<< NavBarAC) 1 
+                  ]
+            ]
+          -- , settingSideBarView push state
+        ]
+
+headerView :: forall w. (Action -> Effect Unit) -> ST.TicketingScreenState -> PrestoDOM (Effect Unit) w
+headerView push state =
+  linearLayout
+    [ width MATCH_PARENT
+    , height WRAP_CONTENT
+    , background Color.black900
+    , orientation VERTICAL
+    , padding $ Padding 16 20 16 25
+    ]
+    [ linearLayout
+        [ width WRAP_CONTENT
+        , height WRAP_CONTENT
+        , gravity CENTER_VERTICAL
+        , onClick push $ const $ BackPressed
+        ]
+        [ imageView
+            [ width $ V 20
+            , height $ V 20
+            , imageWithFallback $ fetchImage FF_ASSET "ic_chevron_left_white"
+            ]
+        , imageView
+            [ width $ V 78
+            , height $ V 42
+            , margin $ MarginLeft 5
+            , imageWithFallback $ fetchImage FF_ASSET "ny_ys_logo_text"
+            ]
+        ]
+    , linearLayout
+        [ width MATCH_PARENT
+        , height WRAP_CONTENT
+        , margin $ MarginTop 29
+        , gravity CENTER_VERTICAL
+        ]
+        [ textView
+            $ [ text "Yatri Sathi Ticketing"
+              , color Color.white900
+              , weight 1.0
+              , height WRAP_CONTENT
+              , gravity CENTER_VERTICAL
+              ]
+            <> FontStyle.subHeading1 LanguageStyle
+        , linearLayout
+            [ width WRAP_CONTENT
+            , height WRAP_CONTENT
+            , cornerRadius 30.0
+            , padding $ Padding 8 8 8 8
+            , gravity CENTER_VERTICAL
+            , background Color.black700
+            , onClick push $ const $ MyTicketsAC
+            , visibility if state.props.hideMyTickets then GONE else VISIBLE
+            ]
+            [ textView
+                $ [ text "My Tickets"
+                  , color Color.white900
+                  , height WRAP_CONTENT
+                  , padding $ Padding 3 0 0 3
+                  ]
+                <> FontStyle.paragraphText LanguageStyle
+            , imageView
+                [ width $ V 16
+                , height $ V 16
+                , imageWithFallback $ fetchImage FF_ASSET "ny_ic_chevron_right_white"
+                ]
+            ]
+        ]
+    ]
+
+ticketingList :: forall w. (Action -> Effect Unit) -> ST.TicketingScreenState -> PrestoDOM (Effect Unit) w
+ticketingList push state =
+  linearLayout
+    [ height MATCH_PARENT
+    , width MATCH_PARENT
+    , orientation VERTICAL
+    , gravity CENTER_HORIZONTAL
+    , margin $ MarginTop 16
+    ] if DA.null state.data.placeInfoArray then [sfl 70, sfl 70] else
+    ( mapWithIndex
+        (\index item -> ticketingItem push item index)
+        state.data.placeInfoArray
+    )
+
+dummyTicketPlaceResp :: API.TicketPlaceResp
```

Comment:
Move it to screen data, and if not required remove

Reply:
done